shared_model/validators/field_validator.hpp

```diff
@@ -35,11 +35,13 @@ namespace shared_model {
      */
     class FieldValidator {
      public:
-      FieldValidator()
+      FieldValidator(unsigned long long future_gap = std::chrono::minutes(5)
```

Comment:
It's better specify the variable width, e.g uint64_t
And in this case (as long as future_gap is something related to time) nice to use time types